index.js

```diff
@@ -27,29 +30,96 @@ const urls = fs
   .split(/\r?\n/)
   .map((l) => `${l}`);
 
-const addonUUID = 'd56a5b99-51b6-4e83-ab23-796216679614';
+let addonUUID = 'd56a5b99-51b6-4e83-ab23-796216679614';
 
-const options = new firefox.Options();
-options.setPreference(
-  'extensions.webextensions.uuids',
-  `{"firefox@ghostery.com": "${addonUUID}"}`,
-);
-// options.addArguments("--headless");
+if (isChromeSelected) {
+  addonUUID = 'agbomdmjjfglgplpnkahbcmblehajkag';
+}
+
+let options = new firefox.Options();
+let selectedBrowser = 'Firefox';
 
-if (isGhosteryEnabled) {
-  options.addArguments('-profile', 'profiles/withGhostery');
-  outputPath += '/withGhostery';
+if (isChromeSelected) {
+  selectedBrowser = 'Chrome';
+  console.log(`LOG: Selected browser: ${selectedBrowser}.`);
+  options = new chrome.Options();
 } else {
-  options.addArguments('-profile', 'profiles/withoutGhostery');
-  outputPath += '/withoutGhostery';
+  console.log(`LOG: Selected browser: ${selectedBrowser}.`);
+  options.setPreference(
+    'extensions.webextensions.uuids',
+    `{"firefox@ghostery.com": "${addonUUID}"}`,
+  );
+  // options.addArguments("--headless");
 }
 
-const outputStream = fs.createWriteStream(`${outputPath}/${timestamp}.txt`);
+let outputPath = 'output/time';
```

Comment:
no need to move this line

Reply:
Extra line added above as far as I can see? Webdriver for Chrome.